README.md

```diff
@@ -97,6 +97,41 @@ In cases where this default variable wrapping can cause problems (e.g. a variabl
 
   `{{#wrap}}[{{nowrap link.text}}]({{nowrap link.url}}){{/wrap}}`
 
+#### Handlebars helpers
+
+In addition to the standard helpers which are part of Handlebars, there are a number of additional helpers defined to assist with Ricardian contract specific issues.
+
+___Since 0.0___
+* `wrap` - Takes no parameters. Block level helper to wrap the contents with `<div>` tags for highlighting. See `Variables` section above for more details.
+
+* `nowrap` - Takes a variable. Indicates that the specified variable *should not* be wrapped with `<div>` tags. See `Variables` section above for more details.
+
+* `symbol_to_symbol_code` - Given a variable containing a 'symbol' string, extract and return only the symbol code. For example: '4,EOS' --> returns 'EOS'.
+
+* `asset_to_symbol_code` - Given a variable containing an 'asset' string, extract and return only the symbol code. For example: '2.001 EOS' --> returns 'EOS'.
+
+___Since 0.1___
+* `account_in_permission_level` - Given a variable containing a permission level object, extract and return the account name.
```

Comment:
It may be good to provide an example like: 'alice@active' ---> returns 'alice'.

Reply:
I'd like to see examples for all of them too...similar to `symbol_to_symbol_code` and `account_in_permission_level`.